packages/app/src/sections/leaderboard/Leaderboard.tsx

```diff
@@ -146,7 +154,8 @@ const Leaderboard: FC<LeaderboardProps> = ({ compact, mobile }) => {
 
 	useEffect(() => {
 		setSearchAddress(searchEns.ensAddress ?? (isAddress(searchTerm) ? searchTerm : ''))
-	}, [searchTerm, searchEns])
+		dispatch(fetchPositionHistoryForTrader(trader))
```

Comment:
I think this needs to be moved to it's own separate useEffect

Reply:
Updated.